app/models/certificate.rb

```diff
@@ -14,17 +14,28 @@ class Certificate < Sequel::Model
   one_to_many :measure_conditions, key: %i[certificate_type_code certificate_code],
                                    primary_key: %i[certificate_type_code certificate_code]
 
+  one_to_many :certificate_type_descriptions, key: %i[certificate_type_code],
+                                              primary_key: %i[certificate_type_code]
+
```

Comment:
This is used by new serializer.

Reply:
All descriptions are fetched by the applicable period that is relevant. We should create this relationship so that it follows the same pattern as certificate_descriptions

Reply:
I think Matt said that is not the case.
So that should be fine.